### PR TITLE
add a new blog of use-KubeKey-to-install-and-deploy-Kubernetes-and-ku…

### PR DESCRIPTION
**Pull request by faweizhao26**
…beovn

Signed-off-by: faweizhao26 <faweizhao@kubesphere.io>

### CODE REVIEW

File: content/zh/blogs/use-KubeKey-to-install-and-deploy-Kubernetes-and-kubeovn.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+keywords: 'KubeSphere, KubeKey, Kubernetes, Kube-OVN, 安装部署'
+description: 'KubeKey 是 KubeSphere V3.0 新增的安装方式，用户可以一键部署 Kubernetes 和 KubeSphere。Kube-OVN 是一款基于 OVS/OVN 的 Kubernetes 网络编排系统。本文将为大家介绍如何使用 KubeKey 来安装部署 Kubernetes 和 Kube-OVN。'
+createTime: '2021-03-16'
+author: '林瑞超'
+snapshot: 'https://pek3b.qingstor.com/kubesphere-community/images/KubeKey-Kubernetes-Kubeovn-banner.png'
+---
+
+> 作者简介：林瑞超，锐捷网络开发工程师， KubeSphere 社区 contributor， 关注 Kube-OVN, Cilium 等容器网络相关技术
+## 背景
+KubeKey 是 KubeSphere V3.0 新增的安装方式，用户可以一键部署 Kubernetes 和 KubeSphere。Kube-OVN 是一款基于 OVS/OVN 的 Kubernetes 网络编排系统。本文将为大家介绍如何使用 KubeKey 来安装部署 Kubernetes 和 Kube-OVN。
```
Comment (FeynmanZhou) [inline content/zh/blogs/use-KubeKey-to-install-and-deploy-Kubernetes-and-kubeovn.md:13]: V3.0 => v3.0

Comment (faweizhao26) [reply content/zh/blogs/use-KubeKey-to-install-and-deploy-Kubernetes-and-kubeovn.md]: Done

**Review by FeynmanZhou — APPROVED** [review]

**Pull request merged**